forecast_solar/__init__.py

```diff
@@ -86,28 +92,22 @@ async def _request(
             self.session = ClientSession()
             self.close_session = True
 
-        try:
-            with async_timeout.timeout(30):
-                response = await self.session.request(
-                    "GET",
-                    url,
-                    params=params,
-                    headers={"Host": "api.forecast.solar"},
-                    ssl=False,
-                )
-                response.raise_for_status()
-        except asyncio.TimeoutError as exception:
-            raise ForecastSolarConnectionError(
-                "Timeout occurred while connecting to Forecast.Solar API"
-            ) from exception
-        except (
-            ClientError,
-            ClientResponseError,
-            socket.gaierror,
```

Comment:
I don't think aiohttp raises socket errors? It's always wrapped in some aiohttp error. It's fine to raise those as-is.